lte/gateway/docker/docker-compose.yaml

```diff
@@ -29,12 +29,12 @@ x-generic-service: &service
 # Generic python anchor to avoid repetition for agw python services
 x-agw-python-service: &pyservice
   <<: *service
-  image: ${DOCKER_REGISTRY}agw_gateway_python:${IMAGE_VERSION}
+  image: ${DOCKER_REGISTRY}agw_gateway_python${OPTIONAL_ARCH_PREFIX}:${IMAGE_VERSION}
```

Comment:
might be my bad, but if it comes in the end it is a `POSTFIX` and not a `PREFIX`?

Reply:
Updated to POSTFIX